### ISSUE TITLE
External Magic Trackpad not Using Driver on Windows 11

### ISSUE THREAD

**Issue by BigPines** [ClementGre/ThreeFingerDragOnWindows#56]
It looks like the driver is registering the internal trackpad on my notebook but does not see the external Apple Magic Trackpad via my USB KVM which is the one I actually want to use. The driver works fine for me on Windows 10 with this same setup. On Windows 11, I see "Touchpad exists and is registered!" but the inputs do not display when I move the external trackpad. That info only shows up when I use the internal trackpad.

![Screenshot 2024-05-06 151010](https://github.com/ClementGre/ThreeFingerDragOnWindows/assets/14079267/ad50b0da-14d3-489a-ac04-2e936b7bee4d)

**Comment by ClementGre** [comment]
This is strange as the app normally subscribes to all of the touchpads at the same time. It should receive inputs from both touchpads.
But you didn't had the issue on Windows 10 with the same configuration right?

**Comment by BigPines** [comment]
Correct. I did not have the issue on Windows 10. It may be because Windows is not recognizing it as a trackpad? In Windows 11, the pointer speed in the Mouse settings controls the pointer speed. On Windows 10, the touchpad settings control that. I also cannot single tap to select nor can I scroll with two fingers like I can in Windows 10 and on the built-in trackpad. I have tried connecting the trackpad via USB and Bluetooth with the same results.
